index.html

```diff
@@ -79,28 +104,28 @@ <h2>How You Can Help</h2>
   <h3>Join Us</h3>
   
   <ul>
-    <li><a href="https://forms.gle/XVdT3JGUbLZQEHXu6">Individual membership application form</a></li>
-    <li><a href="https://forms.gle/yifokDd3rfdFRG7i8">Corporate membership application form</a></li>
-    <li>Not ready to apply? Join the <a href="https://github.com/email-markup-consortium/email-markup-consortium/discussions">discussion on GitHub</a></li>
+    <li><a href="https://forms.gle/XVdT3JGUbLZQEHXu6" target="_blank">Individual membership application form</a></li>
+    <li><a href="https://forms.gle/yifokDd3rfdFRG7i8" target="_blank">Corporate membership application form</a></li>
+    <li>Not ready to apply? Join the <a href="https://github.com/email-markup-consortium/email-markup-consortium/discussions" target="_blank">discussion on GitHub</a></li>
   </ul>
   
   <h3>Spread the Word</h3>
   
   <ul>
-    <li><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Femailmarkup.org">Twitter</a></li>
-    <li><a href="https://www.linkedin.com/shareArticle?mini=true&url=https%3A%2F%2Femailmarkup.org">LinkedIn</a></li>
-    <li><a href="https://www.facebook.com/sharer?u=https%3A%2F%2Femailmarkup.org">Facebook</a></li>
+    <li><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Femailmarkup.org" target="_blank">Twitter</a></li>
+    <li><a href="https://www.linkedin.com/shareArticle?mini=true&url=https%3A%2F%2Femailmarkup.org" target="_blank">LinkedIn</a></li>
+    <li><a href="https://www.facebook.com/sharer?u=https%3A%2F%2Femailmarkup.org" target="_blank">Facebook</a></li>
   </ul>
   
   <h2>Supporters</h2>
   
   <p>The Email Markup Consortium and its mission are supported by employees of Salesforce, Squarespace, Litmus, Email on Acid, SparkPost, Constant Contact, ActionRocket, Parcel, GitHub, and more.</p>
   
   <footer>
-    <ul>
-      <li><a href="https://forms.gle/1Qra9paKh8Ukm2NT7">Contact us</a></li>
-      <li><a href="https://twitter.com/emailmarkup">Twitter</a></li>
-      <li><a href="https://github.com/email-markup-consortium">GitHub</a></li>
+    <ul class="social-nav">
+      <li><a href="https://forms.gle/1Qra9paKh8Ukm2NT7" target="_blank"><img src="img/mail-icon.png" alt="contact us"></a></li>
+      <li><a href="https://twitter.com/emailmarkup"target="_blank"><img src="img/twitter-icon.png" alt="visit us on twitter"></a></li>
+      <li><a href="https://github.com/email-markup-consortium"target="_blank"><img src="img/github-icon.png" alt="go to our github repo"></a></li>
```

Comment:
Currently there's no `img/` directory. Do we have these icons?

Reply:
Sorry, img/ directory now added with icon images for github-icon, twitter-icon, mail-icon, and new-tab-icon